docs/repo-rpm.md

```diff
@@ -0,0 +1,60 @@
+# RPM repository server
+
+| | repo-rpm01.osuosl.theforeman.org |
+| - | - |
+| type | OpenStack VM |
+| OS | CentOS Stream 9 |
+| CPUs | 2 |
+| RAM | 4GB |
+| Storage | /dev/sda (30GB): root, /dev/sdb (100GB): data (LVM) |
+| Managed by | [rpm.pp](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/profiles/manifests/repo/rpm.pp) |
+
+## Domains
+
+These domains are all hosted on the server.
+
+* rpm.theforeman.org
+* stagingrpm.theforeman.org
+
+### Fastly CDN
+
+A Fastly CDN exists that sits in front of:
+
+* rpm.theforeman.org
+* stagingrpm.theforeman.org
+
+For these, the server acts as a backend while the content is served from the Fastly CDN to users.
```

Comment:
Does the rpm server also have the backends concept?

Reply:
Yes.